<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="schedule_tbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>541</width>
     <height>257</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>8 am</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12 pm</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Monday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Thursday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Friday</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>course1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>course2</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>course1</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>course3</string>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>course3</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="look_ahead_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Look Ahead</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_schedule_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Schedule</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_me_lbl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ScheduleMe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recommendations_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Recommendations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preferences_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
